Update to Apex status - clarity on status (#3119)
forcedotcom/salesforcedx-vscode: docs/_articles/en/user-guide/perf-enhancements.md

|                         | Library complete	| Available to try in VS Code	| On by default in VS Code	| CLI Updated  |
|-------------------------|:-----------------:|:---------------------------:|:-------------------------:|:------------:|
| Apex Library	          |        ✔️          |               ✔️             |             ✔️             |       ✔️        |
| Deploy Retrieve Library |        ✔️          |               ✔️             |             ✔️              |              |
	

## Setup

> There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).  Setting to false means VS Code will call the CLI to execute those commands. 

Reviewer: ```suggestion
There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).  Setting to false means that VS Code uses the CLI to execute those commands. 
```

Reviewer: @smaddox-sf - Does "uses the CLI" work?
